planning/behavior_path_planner/src/behavior_path_planner_node.cpp

```diff
@@ -495,46 +493,36 @@ BehaviorTreeManagerParam BehaviorPathPlannerNode::getBehaviorTreeManagerParam()
   return p;
 }
 
-void BehaviorPathPlannerNode::waitForData()
+// wait until mandatory data is ready
+bool BehaviorPathPlannerNode::isDataReady()
 {
-  // wait until mandatory data is ready
-  while (!current_scenario_ && rclcpp::ok()) {
-    RCLCPP_INFO_SKIPFIRST_THROTTLE(get_logger(), *get_clock(), 5000, "waiting for scenario topic");
```

Comment:
@TakaHoribe 
I think these missing info is useful for scenario simulator like this PR
https://github.com/autowarefoundation/autoware.universe/commit/05f6a6bafd545859d09413322def4f94d756a888

Reply:
add info print